themes/1_shell.omp.json

```diff
@@ -69,7 +69,7 @@
         {
           "foreground": "#94ffa2",
           "style": "diamond",
-          "template": " <#ffffff>CPU:</> {{ round .PhysicalPercentUsed .Precision }}% ",
+          "template": " <#ffffff>MEM:</> {{ round .PhysicalPercentUsed .Precision }}% ",
```

Comment:
This does not specify exactly the same thing: the segment below specifies the number of GB available, whereas the percentage specifies the percentage available.

Reply:
@oriionn I would add that to the same segment in that case as this will fetch the information twice, slowing things down unnecessarily. If that’s not possible, use a text segment and cross segment properties.

Reply:
And that's it! I've put together the number of GBs and the percentage into one